sanic/request.py

```diff
@@ -426,28 +426,36 @@ def credentials(self) -> Optional[Credentials]:
                 pass
         return self.parsed_credentials
 
+    def get_form(self, keep_blank_values: bool = False):
```

Comment:
Doesn't have a return annotation. Looks like the function also is missing a return value.